### PR TITLE
[REEF-970] Remove unused files in reef-common

### PR DESCRIPTION
**Pull request by dongjoon-hyun**
JIRA:
  [REEF-970](https://issues.apache.org/jira/browse/REEF-970)

Pull request:
  This closes #

### CODE REVIEW

**Comment by markusweimer** [comment]
I will test and merge.

File: lang/java/reef-common/src/main/conf/log4j.properties [hunk 1]
```diff
@@ -1,40 +0,0 @@
-#
```
Comment (markusweimer) [inline lang/java/reef-common/src/main/conf/log4j.properties]: This file needs to stay, as we use it in some maven profiles, I believe.

**Comment by markusweimer** [comment]
Correction: One of the files needs to stay. I will correct that during the merge.

File: lang/java/reef-common/src/main/conf/log4j.properties (hunk 1 above)
Comment (markusweimer) [inline lang/java/reef-common/src/main/conf/log4j.properties]: I just saw the comment on the JIRA. I will delete this as well.

**Comment by dongjoon-hyun** [comment]
Thank you, @markusweimer .

**Pull request closed without merging**